answers.md

```diff
@@ -0,0 +1,26 @@
+1.	Describe the biggest difference between .forEach & .map.
+
+	.forEach() takes a function and iterates through each array instance, returning the answers individually.
```

Comment:
While not exactly wrong, what we're looking for is how .forEach mutates the original array while .map returns a new array without touching the original.